quipucords/scanner/satellite/six.py

```diff
@@ -229,7 +229,10 @@ def host_fields(scan_task, api_version, url, org_id, host_id):
 
     os_release = host_info.get(OS_RELEASE)
     if os_release:
-        host_info[OS_NAME] = os_release.split(' ')[0]
+        os_name = os_release.split(' ')[0]
```

Comment:
I would check for these prefixes in the following order:
1) Red Hat Enterprise Linux
2) Red Hat
3) RedHat

You can do a `'test string'.lower().startswith('Red Hat Enterprise Linux') `

Reply:
I don't need to check for `RedHat` because it captures it since it is one word.